frontend/app/routes-flow/apply-flow.ts

```diff
@@ -162,7 +200,9 @@ async function start({ id, request }: StartArgs) {
 
   const sessionName = getSessionName(parsedId);
   session.set(sessionName, initialState);
-  session.set('timeUpdated', Date.now());
+
+  const timeUpdatedSessionName = getTimeUpdatedSessionName(id);
+  session.set(timeUpdatedSessionName, new Date().toISOString());
```

Comment:
```suggestion
  session.set(timeUpdatedSessionName, new Date());
```

Reason:
The date will be `JSON.stringify()'d` when storing to session, which will automatically convert it to an ISO string.

Reply:
But it means that I need to parse it as Date when reading it.

Reply:
See <https://github.com/DTS-STN/canadian-dental-care-plan/pull/803#discussion_r1539016895>